.eslintrc

```diff
@@ -0,0 +1,253 @@
+ecmaFeatures: {}
+rules:
+  no-alert: 0
+  no-array-constructor: 0
+  no-bitwise: 0
+  no-caller: 0
+  no-catch-shadow: 0
+  no-class-assign: 0
+  no-cond-assign: 2
+  no-console: 2
+  no-const-assign: 0
+  no-constant-condition: 2
+  no-continue: 0
+  no-control-regex: 2
+  no-debugger: 2
+  no-delete-var: 2
+  no-div-regex: 0
+  no-dupe-keys: 2
+  no-dupe-args: 2
+  no-duplicate-case: 2
+  no-else-return: 0
+  no-empty: 2
+  no-empty-character-class: 2
+  no-empty-label: 0
+  no-eq-null: 0
+  no-eval: 0
+  no-ex-assign: 2
+  no-extend-native: 0
+  no-extra-bind: 0
+  no-extra-boolean-cast: 2
+  no-extra-parens: 0
+  no-extra-semi: 2
+  no-fallthrough: 2
+  no-floating-decimal: 0
+  no-func-assign: 2
+  no-implicit-coercion: 0
+  no-implied-eval: 0
+  no-inline-comments: 0
+  no-inner-declarations:
+  - 2
+  - functions
+  no-invalid-regexp: 2
+  no-invalid-this: 0
+  no-irregular-whitespace: 2
+  no-iterator: 0
+  no-label-var: 0
+  no-labels: 0
+  no-lone-blocks: 0
+  no-lonely-if: 0
+  no-loop-func: 0
+  no-mixed-requires:
+  - 0
+  - false
+  no-mixed-spaces-and-tabs:
+  - 2
+  - false
+  linebreak-style:
+  - 0
+  - unix
+  no-multi-spaces: 0
+  no-multi-str: 0
+  no-multiple-empty-lines:
+  - 0
+  - max: 2
+  no-native-reassign: 0
+  no-negated-in-lhs: 2
+  no-nested-ternary: 0
+  no-new: 0
+  no-new-func: 0
+  no-new-object: 0
+  no-new-require: 0
+  no-new-wrappers: 0
+  no-obj-calls: 2
+  no-octal: 2
+  no-octal-escape: 0
+  no-param-reassign: 0
+  no-path-concat: 0
+  no-plusplus: 0
+  no-process-env: 0
+  no-process-exit: 0
+  no-proto: 0
+  no-redeclare: 2
+  no-regex-spaces: 2
+  no-reserved-keys: 0
+  no-restricted-modules: 0
+  no-return-assign: 0
+  no-script-url: 0
+  no-self-compare: 0
+  no-sequences: 0
+  no-shadow: 0
+  no-shadow-restricted-names: 0
+  no-spaced-func: 0
+  no-sparse-arrays: 2
+  no-sync: 0
+  no-ternary: 0
+  no-trailing-spaces: 0
+  no-this-before-super: 0
+  no-throw-literal: 0
+  no-undef: 2
+  no-undef-init: 0
+  no-undefined: 0
+  no-unexpected-multiline: 0
+  no-underscore-dangle: 0
+  no-unneeded-ternary: 0
+  no-unreachable: 2
+  no-unused-expressions: 0
+  no-unused-vars:
+  - 2
+  - vars: all
+    args: after-used
+  no-use-before-define: 0
+  no-useless-call: 0
+  no-void: 0
+  no-var: 0
+  no-warning-comments:
+  - 0
+  - terms:
+    - todo
```

Comment:
Yeah, this is a boilerplate file that CodeClimate asks for - we can tune it, but I haven't gone in and poked around in great depth.